lib/streamlit/util.py

```diff
@@ -130,9 +130,10 @@ def index_(iterable, x) -> int:
     for i, value in enumerate(iterable):
         # https://stackoverflow.com/questions/588004/is-floating-point-math-broken
         # https://github.com/streamlit/streamlit/issues/4663
-        if isinstance(iterable, np.ndarray):
-            if abs(x - value) < FLOAT_EQUALITY_EPSILON:
-                return i
+        if len(iterable) != 0:
```

Comment:
I'm pretty sure this line doesn't do anything since you'll never be in the for loop body to begin with if `iterable` is empty